### ISSUE TITLE
type conversion bug for filter config when importing record

### ISSUE THREAD

**Issue by mjuarez** [ns1-terraform/terraform-provider-ns1#132]
### Terraform Version
```
Terraform v0.12.29
+ provider.ns1 v1.8.4
```

### Affected Resource(s)
Please list the resources as a list, for example:
- ns1_record

### Terraform Configuration Files
```hcl
resource "ns1_record" "example" {
  zone   = "example.xyz"
  domain = "huh.example.xyz"
  type   = "CNAME"
  ttl    = 3600
}
```

### Debug Output
https://gist.github.com/mjuarez/3d20f0efc53cf710d4265bb112735c11

### Expected Behavior
I should be able to import an ns1_record

### Actual Behavior
I receive an error about expecting string for a filter config, when I received a float64.

```Error: [DEBUG] Error setting filters for: huh.example.xyz, error: &errors.errorString{s:"filters.1.config.N: '' expected type 'string', got unconvertible type 'float64'"}```

### Steps to Reproduce
1. Create a record in a zone with a "Select First N" filter configured for 1 answer.
2. `terraform import ns1_record.example example.xyz/huh.example.xyz/CNAME`

### Important Factoids
I've replaced our domain with example.xyz

### References
Potentially related to:
- #30 
- #31
- https://gist.github.com/tenzone/2fb49bb39bb341ef9c8deff50822e927#file-crash-log-L1221

**Comment by liebman** [comment]
I was literally just about to post this same issue.  It seems that the NS1 api will sometimes return a string:
```
    {
      "filter": "select_first_n",
      "config": {
        "N": "1"
      }
    }
```
and sometimes return as a number:
```
    {
      "filter": "select_first_n",
      "config": {
        "N": 1
      }
    }
```
These are from two different records in one zone and due to this issue any records returning in number format can't be imported.

**Comment by Zach-Johnson** [comment]
Hey @mjuarez and @liebman - thanks for bringing this to our attention and for submitting the fix for it. I've merged the PR. However, we'll likely not cut a release of the provider until later this week as we're in the process of getting this provider migrated into Hashicorps new provider repository.

**Comment by Zach-Johnson** [comment]
Hey @mjuarez @liebman just circling back on this - this is released on v1.8.5 of the provider, on Hashi's new registry: https://registry.terraform.io/providers/ns1-terraform/ns1/latest

**Comment by liebman** [comment]
Looking at the repo that release/tag is pointing to, yes.
